pkg/chunkenc/unordered_test.go

```diff
@@ -268,7 +267,7 @@ func BenchmarkHeadBlockWrites(b *testing.B) {
 	// unordered, unordered
 
 	// current default block size of 256kb with 75b avg log lines =~ 5.2k lines/block
-	var nWrites = (256 << 10) / 50
+	nWrites := (256 << 10) / 50
```

Comment:
nit: Move it to `const`?

Reply:
Only used for this benchmark not sure it has big value.